functions/Backup-DbaDbCertificate.ps1

```diff
@@ -240,6 +240,14 @@ function Backup-DbaDbCertificate {
             $InputObject += Get-DbaDbCertificate -SqlInstance $SqlInstance -SqlCredential $SqlCredential -Database $Database -ExcludeDatabase $ExcludeDatabase -Certificate $Certificate
         }
 
+        if ($Certificate) {
+            $missingCerts = $Certificate | Where-Object { $InputObject.Name -notcontains $_ }
+
+            if ($missingCerts) {
+                Write-Message -Level Warning -Message "Database certificate(s) $missingCerts not found in Database(s)=$Database on Instance(s)=$SqlInstance"
```

Comment:
Is this message actually readable if multiple certs are found missing (when user passes in more than one)?

Reply:
Yes, there is a pester test that checks when multiple certs are not found. The warning message has a space between the cert names and looks like: 

"WARNING: [18:49:58][Backup-DbaDbCertificate] Database certificate(s) dbatoolscli_invalidCertName dbatoolscli_invalidCertName2 not found in Database(s)=DatabaseName on
Instance(s)=MachineName\InstanceName"